### DISCUSSION TITLE
180

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#147] [General]
# 潮流周刊第180期 - 田园一角

封面图拍摄来源于新园区足球场旁边的一个小菜园里面的一角，居然有这么一片原生态的菜园，茄子/葫芦瓜/辣椒/西红柿，每个月可去采摘，很自然，以后我老了，想在家旁边弄一块便宜的地，种种菜应该会很有感觉。

https://weekly.tw93.fun/posts/180-%E7%94%B0%E5%9B%AD%E4%B8%80%E8%A7%92

**Comment by kuenwei** [comment]
轻量原生 Mac 代码编辑器 CodeEdit

这一段是否放错了叙述文字？

**Reply by tw93** [reply]
谢谢指出 已经修改

**Reply by kuenwei** [reply]
昨天折腾一晚上后，发现 CodeEdit 现在输入中文还是有很大的 Bug，目前也还在等将新的 language 更新上去，希望能用来编辑 markdown 啊！

**Comment by DBvc** [comment]
OpenSans的链接有误，链接区分了大小写，应该是Open+Sans

**Reply by tw93** [reply]
的确 这个有些坑，看起来是浏览器copy的时候导致，已经修改

**Comment by Beeta** [comment]
我用 alfred，数次尝试切换到 raycast 但是都换回来了。想了想几个原因:
1. raycast 每个步骤都要点一次回车，感觉比较繁琐
2. 不支持自定义热键：比如在 alfred 里我用「\\」来获取当前网页的 md 形式链接，用!+keyword，输出 snippets
3. raycast的插件好像只支持 js/或 ts？但我更习惯用 python、apple script、bash

不知道现在版本的 raycast 怎么样了，上面的情况是否已经可以解决？请大家指教[抱拳]

**Comment by duofu** [comment]
始终认为，ffmpeg是我用过的视频/gif工具里最好最全也最复杂的。与ffmpeg相比，Gifski实在不算什么

**Comment by Beeta** [comment]
关于编辑器，Octarine可以关注下。rust 开发，特别小巧：https://octarine.app/

**Comment by somkanel** [comment]
所以京东保洁是买了两个 2 小时的服务，还是买了一个 4 小时的单次服务（主要想问如何才能派到两个阿姨，感觉一个人做 4 小时不如 2 个人做 2 小时

**Reply by somkanel** [reply]
好的，已经看到了，是 29r/h x 2

**Reply by tw93** [reply]
哈哈 一般是平台会这样安排 可能也是为了整体效率

**Comment by shellGrace** [comment]
好久没来（moyu）了，沉浸在工作中无法自拔QAQ。咸鱼真的绝绝子，3元买酒店早餐券，省钱一绝
